src/lib.rs

```diff
@@ -59,9 +59,10 @@ where
         self.writer.write(iter)
     }
     pub fn flush_with_gamma(&mut self) -> Result<(), T::Error> {
-        let iter = brightness(gamma(
-            self.content.as_slice().iter().cloned()
-        ), self.brightness);
+        let iter = brightness(
```

Comment:
I believe this is only a whitespace change?

Reply:
Hey - I have reviewed the changes.  Thanks for making them.

I believe the first two are just whitespace or ordering changes.  No functional change
The third is valid, I had this locally, but it wasn't in the MR - my bad, sorry!
The fourth is also valid but not a functional change.  It fixes a linting error.